### DISCUSSION TITLE
Getting this error when deploying the script

### DISCUSSION THREAD

**Discussion by heheboii11** [Cyfrin/foundry-full-course-cu#2843] [Q&A] [answered]
I am getting this error related to test but I am actually deploying the scripts.
```ramireddykarthikreddy@Ramireddys-MacBook-Air Foundry-StableCoin-Hehe % forge script script/DeployDSC.s.sol:DeployDsc --rpc-url $SEPOLIA_URL --private-key $PRIVATE_KEY --broadcast
[⠒] Compiling...
No files changed, compilation skipped
2024-11-10T17:21:25.446206Z ERROR foundry_compilers_artifacts_solc::sources: error="/Users/ramireddykarthikreddy/Foundry-StableCoin-Hehe/test/DscEnginetest.t.sol": No such file or directory (os error 2)
Error: failed to read artifact source file for `/Users/ramireddykarthikreddy/Foundry-StableCoin-Hehe/test/DscEnginetest.t.sol`

Context:
- "/Users/ramireddykarthikreddy/Foundry-StableCoin-Hehe/test/DscEnginetest.t.sol": No such file or directory (os error 2)
ramireddykarthikreddy@Ramireddys-MacBook-Air Foundry-StableCoin-Hehe % 
```

I dont know the exact error that I am facing, please help me to find out with the solution.

**Comment by EngrPips** [comment, marked as answer]
Hello @heheboii11, Please run `forge clean` and delete your `cache` folder, then rerun the script, and let's see how things play out.

**Reply by heheboii11** [reply]
It actually did work, funny how I suggested to do the same to others. But it did not came into my mind, when I face the same type of issue. Anyways, thanks for being here and helping out everyone fam!!

**Reply by EngrPips** [reply]
That is Life for you, my brother. We always know the solution to others problems but our own problems become difficult to comprehend.

**Reply by stugeria** [reply]
Thanks. This helped me with my Deploy script error instantly. 👏

**Reply by EngrPips** [reply]
Awesome, I am glad to hear it helped you. Keep crushing it.
